pg-migration/test/pg/migration_test.clj

```diff
@@ -123,7 +123,31 @@
             mig/parse-url)]
 
     (is (= nil
-           (dissoc res :url)))))
+           (dissoc res :url))))
+
+  (testing "no leading path"
+    (let [res
+          (-> "003.hello.prev.sql"
+              io/file
+              io/as-url
+              mig/parse-url)]
+
+      (is (= {:id 3
+              :slug "hello"
+              :direction :prev}
+             (dissoc res :url)))))
+
```

Comment:
Does the result of this case look good? @jeaye

Reply:
Looks great! I've tested this locally and it fixed the issue. Thank you!

After merging, if you wouldn't mind publishing a new release for this, that would be superb.

Reply:
Yep, will do tomorrow.